package.json

```diff
@@ -4,7 +4,7 @@
   "dependencies": {
     "chai": "4.2.0",
     "fecha": "https://github.com/taylorhakes/fecha/archive/91680e4db1415fea33eac878cfd889c80a7b55c7.tar.gz",
-    "npm-test-project": "git+https://gitlab.com/eakhmano-test-group/eakhmano-test-subgroup/subsubgroup/npm-test-project.git",
-    "test-npm-dep": "git+https://elinah@bitbucket.org/elinah/test-npm-dep.git"
+    "is-positive": "https://github.com/kevva/is-positive.git",
```

Comment:
Can we create a fork for that one?

Reply:
Why do you need a fork?

Reply:
The repo could be moved anytime. We can not rely on someones personal repository. No trustful organization moderates it.

Reply:
It will be better, we should do that with all github dependencies we have. Although it will require a lot of changes.

Reply:
There is `fecha` some lines above that is also a personal repo. It seems to be a fairly popular library. I think that's how the NPM environment is? I wouldn't bother too much with removing those.

Did you add `is-positive` to have another dependency in the place of those that are getting removed, Elina?

Reply:
I don't think we want to maintain many other repos. We don't use them and in case of problem we will just replace them with something small and simple.